### PR TITLE
split module 63

### PR DESCRIPTION
**Pull request by red031000**

### CODE REVIEW

File: arm9/modules/63/asm/mod63_021D74E0.s [hunk 1]
```diff
@@ -0,0 +1,2534 @@
...
+	add r6, r1, #0
+	add r4, r2, #0
+	bl MOD63_021D86E0
+	add r0, r6, #0
+	add r1, r4, #0
+	add r2, r5, #0
+	bl MOD63_021D83A8
+	ldr r0, _021D7D5C ; =gGameVersion
+	ldrb r0, [r0]
+	cmp r0, #0xa
```
Comment (PikalaxALT) [inline arm9/modules/63/asm/mod63_021D74E0.s:903]: ` ; VERSION_DIAMOND`

File: arm9/modules/63/asm/mod63_021D74E0.s [hunk 2]
```diff
@@ -0,0 +1,2534 @@
...
+	str r0, [sp, #8]
+	mov r0, #0x30
+	mov r1, #0xc
+	add r2, r5, #0
+	mov r3, #3
+	str r4, [sp, #0xc]
+	bl FUN_020068C8
+	ldr r0, _021D85C8 ; =gGameVersion
+	ldrb r7, [r0]
+	cmp r7, #0xa
```
Comment (PikalaxALT) [inline arm9/modules/63/asm/mod63_021D74E0.s:1916]: `; VERSION_DIAMOND`

File: arm9/modules/63/asm/mod63_021D74E0.s [hunk 3]
```diff
@@ -0,0 +1,2534 @@
...
+	sub sp, #0x10
+	add r2, #0xa8
+	add r7, r0, #0
+	ldrh r0, [r2]
+	add r5, r1, #0
+	cmp r0, #0
+	bne _021D863A
+	ldr r0, _021D8640 ; =gGameVersion
+	ldrb r0, [r0]
+	cmp r0, #0xa
```
Comment (PikalaxALT) [inline arm9/modules/63/asm/mod63_021D74E0.s:2134]: `; VERSION_DIAMOND`

File: arm9/modules/63/asm/mod63_021D74E0.s [hunk 4]
```diff
@@ -0,0 +1,2534 @@
...
+	thumb_func_end MOD63_021D8650
+
+	thumb_func_start MOD63_021D86E0
+MOD63_021D86E0: ; 0x021D86E0
+	push {r3, r4, r5, lr}
+	sub sp, #0x18
+	add r4, r0, #0
+	ldr r0, _021D8858 ; =gGameVersion
+	ldrb r0, [r0]
+	cmp r0, #0xa
```
Comment (PikalaxALT) [inline arm9/modules/63/asm/mod63_021D74E0.s:2276]: `; VERSION_DIAMOND`

**Comment by red031000** [comment]
done

**Pull request merged**